assets/js/ucc_webrtc.js

```diff
@@ -36,13 +36,13 @@ require('./device_manager');
       console.log('UcxUcc', UcxUcc);
       console.log('Mscs', UcxUcc.Mscs);
       var dm = false;
-      console.log('DeviceManager', UcxUcc.DeviceManager);
-      if (UcxUcc.DeviceManager) {
+      console.log('DeviceManager', UccChat.DeviceManager);
+      if (UccChat.DeviceManager) {
       console.log('dm 1', dm);
-        dm = UcxUcc.DeviceManager;
+        dm = UccChat.DeviceManager;
       } else if (UcxUcc.Mscs) {
```

Comment:
This isn't correct. There should be no references do Mscs here in the master version. This is why we created a plugin approach.